### ISSUE TITLE
Why are styles used in state?

### ISSUE THREAD

**Issue by nushydude** [CodeLanka/ez-net-app#13]
I noticed that in the App.js and the DetailScreen.js, styles are being used as the component state.

Why are they in state??? 

Do the styles ever get `changed` on the fly? Even if they did, they should be `props` passed down from a root level component.

```
export default class DetailScreen extends Component {
    state = {                              <----------------------- defined here
        flexDirection: 'row',
        justifyContent: 'center',
        alignItems: 'center',
    }
    render() {
        
        const {flexDirection, alignItems, justifyContent} = this.state
        const layoutStyle = {flexDirection, justifyContent, alignItems}   <------- and it's being extracted here
        
        return (
          <Container>

              <Content>
                <View style={[styles.layout, layoutStyle]}>   <------------------- and used here
```

**Comment by chathuralakmal** [comment]
@nushydude i wrote those parts. im new to React-Native :) and i may have copied it from tutorial. ill remove it from state.

**Comment by nushydude** [comment]
@chathuralakmal Everyone's new to React Native. :)

One more thing. The style should be like `style={styles.layout}`, and not `style={[styles.layout]}` although the latter might still work. (I haven't checked.) It should be an object, not an array.

Check here: (search by `style`)
https://facebook.github.io/react/docs/dom-elements.html

**Comment by agentmilindu** [comment]
Yes, we have to refactor them in a better way. To be honest,  even I am new to React Native, thus no worries, we learn from each other :D  

I already did some changes in the last commit, may be you two can work on the rest.

**Comment by agentmilindu** [comment]
Fixed by #14

**Comment by nushydude** [comment]
I know this issue is closed, but just needed to update on a bit of information. 

It seems using an `array` as the style prop is a pattern. But that second one should come from `props`. It should not beed `state`.
